tests/cmdline/commands/test_setup.py

```diff
@@ -19,6 +19,9 @@
 from aiida.manage.external.postgres import Postgres
 from pgtest.pgtest import PGTest
 
+# TODO: Make these tests work with sqlite backend
```

Comment:
Meh, these commands are deprecated anyway and the commands themselves only setup profiles that require PSQL, so it only makes sense that this doesn't work with SQLite, does it?